libbpf-cargo/src/gen.rs

```diff
@@ -490,29 +496,31 @@ fn gen_skel_contents(_debug: bool, obj: &UnprocessedObj) -> Result<String> {
         "#
     )?;
 
-    let mut obj_file_path = obj.out.clone();
-    obj_file_path.push(format!("{}.bpf.o", obj.name));
-
     write!(
         skel,
         r#"
         const DATA: &[u8] = include_bytes!("{}");
         "#,
-        obj_file_path.as_path().display()
+        obj_file_path.display()
     )?;
 
     // Capitalize object name
     let mut obj_name = String::new();
-    // Unwrap is safe b/c already checked that `obj.name` contains chars
-    obj_name += &obj.name.chars().next().unwrap().to_uppercase().to_string();
-    if obj.name.len() > 1 {
-        obj_name += &obj.name[1..];
+    // Unwrap is safe b/c already checked that `raw_obj_name` contains chars
+    obj_name += &raw_obj_name
+        .chars()
+        .next()
+        .unwrap()
+        .to_uppercase()
+        .to_string();
+    if raw_obj_name.len() > 1 {
+        obj_name += &raw_obj_name[1..];
```

Comment:
you've added capitalize_first_letter in https://github.com/libbpf/libbpf-rs/pull/43, so once that one is merged you can use it here, right?